wiki/Manual.md

```diff
@@ -4855,6 +4855,8 @@ $order = $exchange->create_order ($symbol, $type, $side, $amount, $price, $param
 ```
 <!-- tabs:end -->
 
+For exchanges, where it is not possible to use attached SL &TP, after submitting an entry order, you can immediatelly submit another order (even though position might not be open yet) with `triggerPrice` and `reduceOnly: true` params, so it can still act as a stoploss order for your upcoming position (note, this approach might not work for some exchanges).
```

Comment:
This might be a bit confusing because when I read attached tp+sl I thought you were referring to type 3 and not type 2.

Reply:
yes, you thought correctly, i meant type 3.

Reply:
for some exchanges, eg biinance you can create trigger reduceOnly order even when you dont have open position. so, even though with API you cant submit T3 order, you can "imitate" an immediate entry + SL order  (with 2 separate orders).